runtime/tests/checker/capability_controller_test.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
Could you please also add a test which asserts types of the type's members? e.g. for `issueHeight` it could be something simple as `let issueHeight: UInt64 = controller.issueHeight`